lib/bundle.js

```diff
@@ -339,14 +348,18 @@ function buildWatch(files, invalidate, build) {
       building = true;
       rebuild = false;
       
-      if (filepath) 
-        ui.log('ok', 'File `' + file + '` changed, rebuilding...');
-      else
+      if (filepath) {
+        if(relFiles.indexOf(filepath) == -1) {
+          ui.log('ok', 'File `' + file + '` changed, but not part of the bundle, skipping...');
```

Comment:
I may have misread this, but it looks like it would still be running the build though in this case?